api/src/main/java/com/alibaba/nacos/api/naming/pojo/ServiceInfo.java

```diff
@@ -48,6 +48,8 @@
     
     private long cacheMillis = 1000L;
     
+    private long failCount = 0L;
```

Comment:
fail count should not be attribute of ServiceInfo. It should be maintained in UpdateTask.